rules/ml/ml_linux_system_user_discovery.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.5.0"]
```

Comment:
```suggestion
ecs_version = ["1.6.0"]
```

Reply:
isn't this intended to indicate the minimum required version? they were tested in 1.5 so they work in either, 1.5 is the min.